api-flask/app/kobo.py

```diff
@@ -41,7 +41,7 @@ def get_kobo_params():
 
     geom_field = request.args.get('geomField')
     if geom_field is None:
-        raise BadRequest('Missing parameter: geomField')
+        logger.debug('Parameter geomField is not set.')
```

Comment:
Actually no, since we don't need the geomField in the frontend anymore it is not strictly required. See Amit's use-case above.

Otherwise we default to "geopoint" further down the line, but we could make this more explicit here